cmd/security-proxy-setup/main.go

```diff
@@ -34,15 +34,15 @@ func main() {
 		usage.HelpCallbackSecurityProxy()
 	}
 	var initNeeded bool
-	var ensureSkipVerify bool
+	var insecureskipverify bool
 	var resetNeeded bool
 	var useProfile string
 	var userTobeCreated string
 	var userOfGroup string
 	var userToBeDeleted string
 	var useRegistry bool
 
-	flag.BoolVar(&ensureSkipVerify, "insureskipverify", true, "skip server side SSL verification, mainly for self-signed cert")
+	flag.BoolVar(&insecureskipverify, "insureskipverify", false, "skip server side SSL verification, mainly for self-signed cert")
```

Comment:
this is still a typo

also not camel case like for security-secretstore-setup where it is `insecureSkipVerify`

Reply:
fixed.

Reply:
Fixed.

Reply:
Fixed.